src/style.scss

```diff
@@ -13,6 +16,15 @@
   text-align: left;
 }
 
+:host {
+  auro-dropdown {
+    &::part(trigger) {
+      min-height: #{$trigger-height};
+      max-height: #{$trigger-height};
```

Comment:
A couple of things here. 

1. When using Sass variables you do not need to use the `#{}` syntax. Referencing the `$trigger-height` variable itself would work. 
2. Let's update this to `var(--trigger-height);`